src/main/kotlin/no/nav/tilleggsstonader/sak/behandling/TestSaksbehandlingService.kt

```diff
@@ -0,0 +1,72 @@
+package no.nav.tilleggsstonader.sak.behandling
+
+import no.nav.tilleggsstonader.sak.vilkår.VilkårService
+import no.nav.tilleggsstonader.sak.vilkår.VilkårStegService
+import no.nav.tilleggsstonader.sak.vilkår.domain.Vilkårsresultat
+import no.nav.tilleggsstonader.sak.vilkår.dto.DelvilkårDto
+import no.nav.tilleggsstonader.sak.vilkår.dto.SvarPåVilkårDto
+import no.nav.tilleggsstonader.sak.vilkår.dto.VilkårDto
+import no.nav.tilleggsstonader.sak.vilkår.dto.VurderingDto
+import no.nav.tilleggsstonader.sak.vilkår.regler.RegelId
+import no.nav.tilleggsstonader.sak.vilkår.regler.SluttSvarRegel
+import no.nav.tilleggsstonader.sak.vilkår.regler.SvarId
+import no.nav.tilleggsstonader.sak.vilkår.regler.SvarRegel
+import no.nav.tilleggsstonader.sak.vilkår.regler.Vilkårsregel
+import no.nav.tilleggsstonader.sak.vilkår.regler.vilkårsreglerForStønad
+import org.springframework.stereotype.Service
+import java.util.UUID
+
+@Service
```

Comment:
Kan kanskje ha med denne her også 
```suggestion
@Service
@Profile("!prod")
```